src/components/global/Header/Navigation.tsx

```diff
@@ -1,51 +1,25 @@
-import { Link, useLocation } from 'react-router-dom';
+import React from 'react';
+import HamburgerNavigation from './HamburgerNavigation';
+import NavigationLinks from './NavigationLinks/NavigationLinks';
+import { Link } from 'react-router-dom';
```

Comment:
@hubert-deriv please consider moving this import to line 2 (right after react import) 🙂

Reply:
fixed